gen.treesitter-ng/build.gradle

```diff
@@ -0,0 +1,22 @@
+description = 'GumTree tree generator for tree-sitter-ng.'
+
+ext.isNative = true
+
+dependencies {
+    implementation("org.yaml:snakeyaml:2.0")
+    implementation("io.github.bonede:tree-sitter-c:0.20.5")
+    implementation("io.github.bonede:tree-sitter-cmake:0.4.1")
+    implementation("io.github.bonede:tree-sitter-c-sharp:0.20.0")
+    implementation("io.github.bonede:tree-sitter:0.22.2")
```

Comment:
Maybe this dependency should go just below snakeyaml?

Reply:
Yes, it should. It's the core module of tree-sitter-ng.